app/src/repositories/comment/commentRepository.js

```diff
@@ -0,0 +1,133 @@
+"use strict";
+
+const pool = require("../../config/db");
+
+class CommentRepository {
+  async findById(commentId) {
+    const [rows] = await pool.query(
+      `
+      SELECT * 
+      FROM comments 
+      WHERE id = ?
+      `,
+      [commentId]
+    );
+    return rows[0];
+  }
+
+  async createComment({ episodeId, userId, content, parentId }) {
+    const [result] = await pool.query(
+      `
+      INSERT INTO comments (episode_id, user_id, content, parent_id) 
+      VALUES (?, ?, ?, ?);
+      `,
+      [episodeId, userId, content, parentId]
+    );
+
+    const [newRows] = await pool.query(
+      `
+        SELECT *
+        FROM comments
+        WHERE id = ?;
+        `,
+      [result.insertId]
+    );
+
+    return newRows[0];
+  }
+
+  async updateComment(commentId, { content }) {
+    await pool.query(
+      `
+      UPDATE comments
+      SET content = ?
+      WHERE id = ?;
+      `,
+      [content, commentId]
+    );
+
+    const [updateRows] = await pool.query(
+      `
+      SELECT *
+      FROM comments
+      WHERE id = ?;
+      `,
+      [commentId]
+    );
+
+    return updateRows[0];
+  }
+
+  async deleteComment(commentId) {
+    await pool.query(
+      `
+      DELETE FROM comments
+      WHERE id = ?;
+      `,
+      [commentId]
+    );
+  }
+
+  async findReaction(commentId, userId) {
+    const [rows] = await pool.query(
+      `
+      SELECT * 
+      FROM comment_likes
+      WHERE comment_id = ? AND user_id = ?;
+      `,
+      [commentId, userId]
+    );
+    return rows[0];
+  }
+
+  async insertReaction(commentId, userId, type) {
```

Comment:
위의 createComment와 같이 맞춰주면 좋을거같아요

```suggestion
  async createReaction(commentId, userId, type) {
```